src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -238,6 +244,10 @@ const KeyboardAwareScrollView = forwardRef<
         onMove: (e) => {
           "worklet";
 
+          if (!enableKeyboardHandling) {
+            return;
+          }
```

Comment:
Wouldn't it be easier to add a check in `maybeScroll` function?

Because actually right now scrolling happens not only in `useKeyboardHandler`, but also in `useAnimatedReaction` (text input changes height) and in `onChangeText`, but in all these places we do call a `maybeScroll`. What do you think?

Reply:
I think we can add an additional condition to the `maybeScroll` function to avoid changes when typing text. Also, I think we should leave this condition in `onStart`, `onEnd`, `onMove` events to avoid some calculations. What do you think?